cleanlab/rank.py

```diff
@@ -57,109 +118,328 @@ def order_label_issues(
       The columns must be ordered such that these probabilities correspond to class 0,1,2,...
       `pred_probs` should have been computed using 3 (or higher) fold cross-validation.
 
-    rank_by : str ['normalized_margin', 'self_confidence']
-      Method to order label error indices (instead of a bool mask), either:
-        'normalized_margin' := normalized margin (p(label = k) - max(p(label != k)))
-        'self_confidence' := [pred_probs[i][labels[i]] for i in label_issues_idx]
+    method : {"self_confidence", "normalized_margin", "confidence_weighted_entropy"}, default="normalized_margin"
+      Label quality scoring method. Default is "normalized_margin".
+
+      Letting `k := labels[i]` and `P := pred_probs[i]` denote the given label and predicted class-probabilities
+      for the `i`th datapoint, its score can either be:
+      'normalized_margin' := P[k] - max_{k' != k}[ P[k'] ]
+      'self_confidence' := P[k]
+      'confidence_weighted_entropy' := entropy(P) / self_confidence
+
+      Self-confidence works better for finding out-of-distribution (OOD) examples, weird examples, bad examples,
+      multi-label, and other types of label errors.
+
+      Normalized margin works better for finding class conditional label errors where
+      there is another label in the class that is better than the given label.
```

Comment:
Language here can be more clear/precise as:

```
Let `C = {0,1,...,K}` denote the classification task's specified set of classes. The normalized_margin score works better for identifying class conditional label errors, i.e. datapoints for which another label in C is appropriate but the given label is not. The self_confidence score  works better for identifying alternative label issues corresponding to bad datapoints that are: not from any of the classes in C, well-described by 2 or more labels in C, or generally just out-of-distribution (ie. anomalous outliers).
```

Reply:
Agree, that sounds better. I made this change. Thanks.